eng/scripts/DownloadDevPackages.ps1

```diff
@@ -0,0 +1,50 @@
+param (
+    [Parameter(Mandatory=$True)]
+    [string] $WorkingDirectory,
+    [Parameter(Mandatory=$True)]
+    [string] $NupkgFilesDestination,
+    [string] $NugetSource="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"
+)
+
+. (Join-Path $PSScriptRoot ".." common scripts common.ps1)
+
+$allPackages = Get-AllPkgProperties
+$trackTwoPackages = $allPackages.Where({ $_.IsNewSdk })
+
+Push-Location $WorkingDirectory
+$nugetPackagesPath = Join-Path $WorkingDirectory nugetPackages
+New-Item -Path $WorkingDirectory -Type "directory" -Name "nugetPackages"
+
+$packagesInFeed = Find-Package -Source $NugetSource -AllVersion -AllowPrereleaseVersions
+
+foreach ($package in $trackTwoPackages)
+{
+  $allVersionsInFeed = $packagesInFeed.Where({ ($_.Name -eq $package.Name) })
```

Comment:
May be worth adding this in a helper in language-settings so we can share it across here and smoke-tests.

FYI @ckairen @benbp